source/components/Dropdown/index.js

```diff
@@ -104,42 +133,77 @@ Dropdown.propTypes = {
      * React Component to display as the Dropdown Content
      */
     children: PropTypes.node,
+
     /**
-     * Whether or not dropdown should have a slight drop shadow
+     * Should dropdown content be scrollable
      */
     contentScrollable: PropTypes.bool,
+
     /**
-     * Hides chevron in dropdown toggle
+     * Should dropdown content be left-aligned with the dropdown toggle
      */
     dropdownLeftAligned: PropTypes.bool,
+
     /**
-     * Whether or not dropdown should have a drop shadow (darker than the one produced by hasShadow)
+     * Should dropdown content be right-aligned with the dropdown toggle
      */
     dropdownRightAligned: PropTypes.bool,
+
     /**
-     * Is it a nested dropdown
+     * Whether or not dropdown should have a drop shadow (darker than the one produced by hasShadow)
      */
     hasDarkShadow: PropTypes.bool,
```

Comment:
I didn't see that before - two different props for shadow are not needed - `shadow: PropTypes.oneOf(false, 'light', 'dark])` should be enough

Reply:
https://github.com/Wikia/react-design-system/pull/77#discussion_r235720272